src/poliastro/twobody/orbit.py

```diff
@@ -1024,6 +1024,34 @@ def propagate(self, value, method=mean_motion, rtol=1e-10, **kwargs):
                 new_epoch,
             )
 
+    @u.quantity_input(value=u.rad)
+    def time_to_anomaly(self, value):
+        """ Returns time required to be in a specific true anomaly.
+
+        Parameters
+        ----------
+        value : ~astropy.units.Quantity
+
+        Returns
+        -------
+        tof: ~astropy.units.Quantity
+            Time of flight required.
+        """
+
+        # TODO: Avoid repeating logic with mean_motion?
+        p, ecc, inc, raan, argp, _ = rv2coe(
```

Comment:
Are we using these values anywhere?

Reply:
Sure! Sorry, did not notice this :sweat: